src/reports/package/package-report-footer.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NamedFC } from 'common/react/named-fc';
+import { serviceTitle, toolName } from 'content/strings/application';
+import { SectionProps } from 'reports/components/report-sections/report-section-factory';
+import { ToolLink } from 'reports/components/report-sections/tool-link';
+
+export type ReportFooterProps = Pick<SectionProps, 'environmentInfo'>;
+
+export const PackageReportFooter = NamedFC<ReportFooterProps>('ReportFooter', ({ environmentInfo }) => {
```

Comment:
This has a lot of duplication from `ReportFooter` (under `src\reports\components\report-sections\report-footer.tsx`). I'm wondering if it's better to change the props in a way that we can reuse the component but pass different props to customize, let's say, the tool name.

Reply:
I will look into that. I dealt with the ToolLink but there is other duplication that we can address.

Reply:
Refactored the duplication away.